### PR TITLE
Add support for multiple melange configs per image

### LINKED ISSUE #137: Support for multiple melange configs per single image
Related to https://github.com/chainguard-images/images/issues/118

### PR DESCRIPTION
**Pull request by jdolitsky**
Resolves #137

### CODE REVIEW

File: images/apko/image.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 versions:
   - melange:
-      config: configs/latest.melange.yaml
+      configs:
+        - configs/latest.melange.yaml
```
Comment (luhring) [inline images/apko/image.yaml:4]: Just a quick question — if we're deciding to support both `config` and `configs` at the same time, why are we changing these instances where there's just a single config to use the multi-config option?

Comment (jdolitsky) [reply images/apko/image.yaml:4]: the `multi-config` option supports a comma-separated list. In the case of a single melange file, this is still a valid list (with just a single item)

We could leave this as `config: configs/latest.melange.yaml`, but I thought it'd be easier for people copy-pasting later who need multiple melange configs to be consistent everywhere

Comment (jdolitsky) [reply images/apko/image.yaml:4]: nevermind actually.. I just dropped support for the single config altogether since it isnt being used anywhere

Comment (luhring) [reply images/apko/image.yaml:4]: Nice!

**Review by luhring — APPROVED** [review]

**Pull request merged**